nbclassic/notebookapp.py

```diff
@@ -274,6 +274,16 @@ def initialize_handlers(self):
                 {"url": self.serverapp.base_url+"files/{0}"}
             )
         )
+        # Add a redirect from /nbclassic to /nbclassic/tree
+        # if both notebook>=7 and nbclassic are installed.
+        if url_prefix_notebook() == "/nbclassic":
+            handlers.append(
+                (
+                    rf"/{self.serverapp.base_url}/nbclassic",
```

Comment:
This line should only be rf"/nbclassic" the base_url is already factored out here. Also which ever one merges second will have to change because I renamed url_prefix_notebook in my merge request.
```python
if url_prefix_notebook().length>0:
    handlers.append(
        (
            rf"/nbclassic",
            RedirectHandler,
            {"url": self.serverapp.base_url+"nbclassic/tree"}
        )
    )
```

Reply:
Thx. https://github.com/jupyter/nbclassic/pull/165 is merged. I have pushed an update that takes your proposal.